qemu-shutdown

```diff
@@ -8,3 +18,12 @@ while [ -S /qemu-monitor ]
 do
   sleep 0.1
 done
+
+echo
+echo
+echo
+echo start shutdown
+/k8s-kvm --set-node-status-to="ShutDown"
+echo end shutdown
+echo
+echo
```

Comment:
> The problem I have right now is that I think the wrapping is done and executed properly but I do not see anything in the logs.

You mean logs like those above?

Reply:
Yes, how can I make this visible? It is not shown in the container logs. I got a brainwave that there is something like a termination log. Is this maybe just written to some special file within the container?

Reply:
IDK :( Looks like kubernetes specific. Maybe ask in #kubernetes.

Reply:
I was thinking about the sidecar but couldn't wrap my head around how to invoke the qemu specific commands we have in this container. How should that look like?

Reply:
If you wrap the original docker-entrypoint in go you could add a http handler for the hook. This way this all gets handled in one main process. Does that make sense?